.github/workflows/assign-reviewers.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Assign Reviewers'
+on:
+  # pull_request_target is suggested for projects where pull requests will be
+  # made from forked repositories. If pull_request is used in these cases,
+  # the github token will not have sufficient permission to update the PR.
+  pull_request_target:
+
+jobs:
+  assign:
+    runs-on: ubuntu-latest
+    name: Assign Reviewers
+    steps:
+      - uses: dyladan/component-owners@main
```

Comment:
I think we should ask for a release (e.g. `1.0.0`) so that it won't magically start failing

Reply:
https://github.com/dyladan/component-owners/issues/13

Reply:
If this PR gets merged - please create an issue to fix it on our side as well.